src/main/scala/com/gd/twitteranalytics/util/AppConfigReader.scala

```diff
@@ -4,19 +4,25 @@ import com.typesafe.config.{Config, ConfigException, ConfigFactory}
 
 object AppConfigReader{
 
-  val appConfig:Config = ConfigFactory.load("application.conf")
-  val twitterConfig:Config = ConfigFactory.load("twitterAuthKeys.conf")
+  @throws[ConfigException]
+  def getAppConfigurables : Array[String] = {
 
-  val SavePath = appConfig.getString("application.save_path")
-  val OutputDateFormat = appConfig.getString("application.date_format")
-  val TweetslangFilter = appConfig.getString("application.filter")
-  val HashTags = appConfig.getString("application.hashtags")
+    val appConfig: Config = ConfigFactory.load("application.conf")
+    val SavePath = appConfig.getString("application.save_path")
+    val TweetsLangFilter = appConfig.getString("application.filter")
+    val HashTags = appConfig.getString("application.hashtags")
+    val MasterUrl = appConfig.getString("application.masterUrl")
+    Array(SavePath,TweetsLangFilter,HashTags,MasterUrl)
+  }
+@throws[ConfigException]
+  def getTwitterAuthKeys : Array[String] = {
 
-  val AppName = appConfig.getString("application.app_name")
-  val MasterUrl = appConfig.getString("application.masterUrl")
+    val twitterConfig: Config = ConfigFactory.load("twitterAuthKeys.conf")
+    val ConsumerKey = twitterConfig.getString("configKeys.twitter4j.oauth.consumerKey")
```

Comment:
please add a TODO to utilize an approach when a config model defined via a case class (e.g. https://pureconfig.github.io/docs/). It will reduce the boilerplate code.

Reply:
Done